### PR TITLE
Adds general-purpose cooking oil, replaces usages of corn oil

### PR DESCRIPTION
**Pull request by Heroman3003**
Adds Cooking Oil - separate reagent from Corn Oil that is produced either via 10 units Corn Oil + 5 units of UE as catalyst or via 10 units of Peanut Oil + 5 units of UE as catalyst

Replaces most usages of Corn Oil in recipies, food and drinks alike, with Cooking Oil. Roasted seed recipies can take any kind of oil though.
Replaces contents of bottles in vending machines, lockers and orderable tank in cargo with Cooking Oil

Changes beer recipe from 10 units of Corn Oil + 5 units of UE as catalyst to 5 units of Corn Oil + 5 units of Flour + 5 units of UE as catalyst.

Basically, this makes corn oil and peanut oil more equal, while also making the nitroglycerine bombmaking actually have to rely on botany or xenobotany instead of just ordering a tank of free corn oil.

Potential todo idea later if this is merged: sunflower seed oil as another oil option, with conversion to cooking oil recipe for it too.

### CODE REVIEW

**Comment by Hatterhat** [comment]
i too enjoy waiting for a nonexistent (xeno)botanist to grow corn for my fire and forget rapid deep ore extraction methods /s

**Comment by FartMaster69420** [comment]
Nitro bombs are in no way necessary for mining. You've got mining charges that can be upgraded, stationary drills, toxins bombs and surface bores for mechs. This change just stops it from being a ten minute affair to blast mine a planet.

**Comment by Heroman3003** [comment]
> 
> 
> i too enjoy waiting for a nonexistent (xeno)botanist to grow corn for my fire and forget rapid deep ore extraction methods /s

Deep ore extraction being "fire and forget rapid" is precisely part of reason behind this PR, as its 100% not meant to be so overtrivialized
